README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for first CFSS homework assignment
+
+#About Me
+Hi! My name is **Di Zhou**. I'm a MAPSS student concentrating in sociology. My research interests are political socialization and social change. 
+
+I have three cats at home! They are:
+* Mountain
```

Comment:
3. I think this is supposed to be use of bullet points, but when I clone your repo and run it in R, it shows up as just asterisks in between your cats name as opposed to bullet point. Maybe you need a space in between the asterisk and the cat name? I'm not sure, because when I compare it to my use of them, which do show up as bullet points, it looks like the exact same code. ALSO when I view your markdown on Git, they do show up as bullet points! So I'm not really sure ... but good!